.github/workflows/build_dynamo_samples_net8.0.yml

```diff
@@ -0,0 +1,60 @@
+# Build DynamoAll.sln with .NET 8.0
+name: Build DynamoSamples.sln net8.0
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+ build:
+  runs-on: windows-latest
+  steps:
+    - name: Checkout DynamoSamples Repo
+      uses: actions/checkout@v4
+      with:
+        path: DynamoSamples
+        repository: DynamoDS/DynamoSamples
+    - name: Setup dotnet
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: '8.0.x'
+    - name: Disable problem matcher
+      run: echo "::remove-matcher owner=csc::"
+
+    - name: Install dependencies for windows runtime
+      run: |
+        dotnet restore $Env:GITHUB_WORKSPACE\DynamoSamples\src\DynamoSamples.sln /p:Configuration=Release --runtime=win-x64 -p:DotNet=net8.0
+    - name: Build DynamoSamples with MSBuild for Windows
+      run: |
+        echo "***Continue with the build, Good luck developer!***"
+        cd "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\"
+        .\MSBuild.exe $Env:GITHUB_WORKSPACE\DynamoSamples\src\DynamoSamples.sln  /p:Configuration=Release /p:DotNet=net8.0
+    # look for DynamoSamples outputs
+    - name: Navigate to dynamo_linter Folder
+      run: |
+        cd "$Env:GITHUB_WORKSPACE\DynamoSamples\dynamo_linter\Sample Linter\bin"
+        echo "***Locating SampleLinter.dll for Windows!***"
+        test ".\SampleLinter.dll" && echo "SampleLinter.dll exists!"
+    - name: Navigate to dynamo_package Folder
+      run: |
+        cd "$Env:GITHUB_WORKSPACE\DynamoSamples\dynamo_package\Dynamo Samples\bin"
+        echo "***Locating dynamo_package dlls for Windows!***"
+        test ".\SampleLibraryUI.dll" && echo "SampleLibraryUI.dll exists!" &&  test ".\SampleLibraryZeroTouch.dll" && echo "SampleLibraryZeroTouch.dll exists!"
+    - name: Navigate to dynamo_viewExtension Folder
+      run: |
+        cd "$Env:GITHUB_WORKSPACE\DynamoSamples\dynamo_viewExtension\Sample View Extension\bin"
+        echo "***Locating SampleViewExtension.dll for Windows!***"
+        test ".\SampleViewExtension.dll" && echo "SampleViewExtension.dll exists!"
+    - name: Save Pull Request Number
+      env:
+        PR_NUMBER: ${{ github.event.number }}
+      run: $env:PR_NUMBER | Out-File -FilePath pr_number.txt
+    - name: Upload Pull Request Artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: pr_number
+        path: ${{ github.workspace }}\pr_number.txt
```

Comment:
what does this end up uploading?

Reply:
The PR_NUMBER I guess see line 53 above (`run: $env:PR_NUMBER | Out-File -FilePath pr_number.txt`)
Not sure why this is needed, I stole it from Dynamo's git workflow

Reply:
I think @avidit may know, it is for the smoke tests follow up, I dont think you need the upload step at all here unless you plan to run UI tests with these?

Reply:
@pinzart90 can you remove it?

Reply:
removed